tests/meta/demultiplex/test_utils.py

```diff
@@ -330,27 +331,27 @@ def test_is_syncing_complete_false(
     assert not is_directory_synced
 
 
-def test_add_flow_cell_name_to_fastq_file_path(bcl2fastq_flow_cell_id: str, fastq_file_path: Path):
+def test_add_flow_cell_name_to_fastq_file_path(bcl2fastq_flow_cell_id: str, fluffy_fastq_file_path):
```

Comment:
Are we sure that these tests used the renamed Path? Unless it is directly imported, I don't think this file can access the fixture.

Reply:
fixed